app/views/users/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="container">
+  <div class="user_info">
+    <div class="img">
+      <%= image_tag @user.photo, alt: "User Photo", class: "user-photo"%>
+    </div>
+    <div class="user">
+      <h2><%= @user.name %></h2>
+      <p><strong>Number of posts: </strong><%= @user.posts.count %></p>
+    </div>
+  </div>
+  <div class="user_bio">
+    <p>Bio</p>
+    <p><%= @user.bio %></p>
+  </div>
+  <% @user.recent_posts.each_with_index do |post, index| %>
+    <div class="post">
+      <h3>Post <%= index+1 %></h3>
+      <p><%= post.title %></p>
+      <p class='count'>Comments: <%= post.comments.count %> | Likes: <%= post.likes.count %></p>
+    </div>
+  <% end %>
+  <%= link_to "See all posts",user_posts_path(@user), class: "button" %> 
+</div>
```

Comment:
- [ ] Kindly ensure the elements stretch to the width of the page like in [the wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png).

  ![image](https://github.com/Obimbo07/tdn/assets/36542986/efd431fd-5214-436a-a1dd-762cf9c39c8e)

Reply:
Fixed. 
![image](https://github.com/Obimbo07/tdn/assets/93598189/b6530f93-1f7b-419f-894f-4aa77a261cfe)